NIPostsReload/Presentation/Scenes/NIPostFeedModule/Presenter/NIPostFeedPresenter.swift

```diff
@@ -81,6 +102,43 @@ final class DefaultNIPostFeedPresenter: NIPostFeedPresenter {
         
         view.updateCollectionViewLayout()
     }
+    
+    func searchPosts(query: String?) {
+        searchWorkItem?.cancel()
+        
+        guard let query, !query.isEmpty else {
+            updatePosts()
+            return
+        }
+        
+        guard query.count >= 2 else {
```

Comment:
If a user types a three-element query and then removes one element, will they still see the same result? This behavior might appear as a bug.